content/05.conclusion.md

```diff
@@ -1,19 +1,28 @@
+We used the full text of documents to correct these missing annotations and our analysis provided a comprehensive understanding of the extent to which preprints are published.
+This correction resulted in a publication rate that is higher than previously estimated.
+Importantly, our analysis only accounts for papers that are published open access, so these results should be considered to raise the lower bound of the total preprint publication fraction; however, the true fraction is likely to be much higher.
+In any case, future work, especially that which aims to assess the fraction of preprints that are eventually published, should account for the possibility of missed annotations.
 
-Previous analyses have focused on article metadata, including which papers are heavily downloaded and discussed [@doi:10.7554/eLife.45133] and by which communities discuss them [@doi:10.1371/journal.pbio.3000860].
-We found that some preprints are high similar to published articles within the PMC open access corpus, and a detailed examination revealed that many preprints were published but not previously annotated.
-Using the full text of documents to correct these missing annotations provides a comprehensive understanding of the extent to which preprints are published and this correction resulted in a publication rate that is higher than previously estimated.
-Importantly, this only accounts for papers that are published open access, so our analysis should be considered to raise the lower bound but the truly published fraction is likely to be higher.
+Our work presents an early step towards understanding the process by which peer review alters life sciences papers.
+We analyzed and compared of the full text of preprints with their corresponding publication.
+Our analysis revealed differences in typesetting and supplementary materials, suggesting that the majority of peer review changes are modest at best. 
+Our results lay out the groundwork for future endeavors that aims to identify sentences and claims that are altered during the peer review process.
+Preprints in general take approximately 16 days on average to venture through the peer review process provided that modest changes are requested.
+We also discovered that the peer review process is agnostic to preprint categories overall with the exception being the scientific communication and education category.
+Future investigation is required to uncover the reasons behind this large delay.
+
+Lastly, we found that document embeddings were associated with the eventual journal at which the work was published.
+We trained two machine learning models to identify which journals publish similar papers towards a query preprint.
+Our models achieved modest results, which proposes that future models should include more information than solely textual content when identifying which journal venues are appropriate for a preprint.

```

Comment:
Why do you argue the results are modest? In terms of fold change over random, they are quite good. I'm not sure what you could expect as a top-line accuracy given that the noise in the system has to be pretty high as it is.